src/shared/components/FluxMonacoEditor.tsx

```diff
@@ -63,6 +65,7 @@ const FluxEditorMonaco: FC<Props> = ({
   const editorDidMount = (editor: EditorType) => {
     prelude.current = setupForReactMonacoEditor(editor)
     prelude.current.updatePreludeModel(variables)
+    setEditor(editor)
```

Comment:
this reduces a bunch of prop drilling and has a graceful failure case if the provider is not defined

Reply:
Like this change.

Note that you missed one in the `src/flows/pipes/RawFluxEditor`.
Once you update that, then the prop `setEditorInstance` can be deleted.